doc/Command-Reference.md

```diff
@@ -5121,6 +5121,25 @@ This command is to display the link-training status of the selected interfaces.
     Ethernet8      trained          on      up       up
   ```
 
+**show interfaces flap**
+
+The show interfaces flap command provides detailed insights into interface events, including the timestamp of the last link down event and the total flap count (number of times the link has gone up and down). This helps in diagnosing stability and connectivity issues.
+
+- Usage:
+  ```
+  show interfaces flap
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ show interfaces flap
+  Interface       Last Link Down          Flap Count  
```

Comment:
Will last link down time stamp be a bit ambiguous? since we don't know when is the counter be calculated without start point, and down time will not show gap between down to up.

Below is some sample from Junos cli maybe referenced:

Physical interface: ge-0/0/1, Enabled, Physical link is Up  
  Interface index: 137, SNMP ifIndex: 515  
  ...  
  Link-level type: Ethernet, MTU: 1514, Speed: 1000mbps, Loopback: Disabled,  
  ...  
  Last flapped   : 2022-10-01 10:00:00 UTC (00:01:23 ago)  
  Statistics last cleared: Never  
  Traffic statistics:  
    Input  bytes  :          1234567890  
    Output bytes  :          9876543210  
    Input  packets:                12345  
    Output packets:                54321  
  ...  
  Input errors: 0, Output errors: 0, Input drops: 0, Output drops: 0  
  ...  
  Protocol eth-switch, MTU: 1514  
  Flags: None  
  ...  
  Link is Up  
  Interface has been up 5 days, 2 hours, 30 minutes, 15 seconds  
  ...  
  Last flap event: None  
  Flap statistics:  
    Since 2022-09-01 00:00:00 UTC  
      Link flaps: 2  
      Up transitions: 3  
      Down transitions: 2

Reply:
@vdahiya12 can you check if we can show the number of up and down events separately? if not we can take that has next enhancement.